.travis.yml

```diff
@@ -12,6 +12,7 @@ install:
 - pip install flake8 flake8-colors flake8-docstrings flake8-import-order
 - pip install pep8-naming
 - pip install pylint
+- pip install psutil
```

Comment:
Please also add `psutil` as a requirement in `setup.py` file

Reply:
@ustinov Done.